npm2deb/templates.py

```diff
@@ -337,3 +337,14 @@
 filenamemangle=s/.*=// \\
  https://qa.debian.org/cgi-bin/fakeupstream.cgi?upstream=npmjs/%(module)s .*=%(module)s-(\d.*)\.(?:tgz|tar\.(?:gz|bz2|xz))
 """
+
```

Comment:
Can you please add gitlab too ?

Reply:
To add GitLab, more than upstream/metadata has to be generated

Reply:
I can add gitlab in a separate MR

Reply:
:+1: